pages/common/vim.md

```diff
@@ -14,6 +14,28 @@
 
 `view {{file}}`
 
-- Exit vim:
+- Switch to normal mode:
 
-`[Esc] (to switch to normal mode), then :q`
+`<Esc>`
+
+- Switch to insert mode from normal mode:
+
+`i`
+
+- Run a command mode from normal mode:
+
+`:{{command}}<Cr>`
+
+- Common commands:
```

Comment:
Something tells me that this would be best as a
```
multiline

block
```

but it didn't pass the linting. Should we consider adding this to the linters rules?